### PR TITLE
Feat/label

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
show notes into label screen 
add filters

### CODE REVIEW

File: src/components/FiltertsModal/filtertsmodal.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.filterts-modal {
+  /* background-color: red; */
```
Comment (tanishk15sharma) [inline src/components/FiltertsModal/filtertsmodal.css:2]: you can remove this

File: src/components/ModalTextEditor/ModalTextEditor.jsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import toast from "react-hot-toast";
+import { TOGGLE_COLOR_PALLETE } from "../../constant/actionTypes";
```
Comment (tanishk15sharma) [inline src/components/ModalTextEditor/ModalTextEditor.jsx:3]: Good to see you are not using `string`

File: src/components/Navbar/navbar.css [hunk 3]
```diff
@@ -5,6 +5,10 @@
   height: 50px;
   background-color: #5348c7;
   padding: 0.5rem;
+  position: sticky;
+  top: 0;
+  left: 0;
+  z-index: 99;
```
Comment (tanishk15sharma) [inline src/components/Navbar/navbar.css:11]: you can give a low `z-index`

**Review by tanishk15sharma — APPROVED** [review]
PR has a lot of changes , naming convention is good

**Review by Manpreet-hub — APPROVED** [review]
Code looks good and well structured.  
Suggestion: Please try to create short PR.

**Pull request merged**